docs/notifications.json

```diff
@@ -11,6 +11,6 @@
   },
   {
     "id": 54,
-    "text": "<a style=\"color: inherit;\" target=\"_blank\" rel=\"noopener\" href=\"https://twitter.com/MaterialUI/status/1438518915236126723\">MUI Core v5 is out 🎉</a>"
+    "text": "<a style=\"color: inherit;\" target=\"_blank\" rel=\"noopener\" href=\"https://twitter.com/MaterialUI/status/1438518915236126723\">MUI Core v5</a> is out 🎉"
```

Comment:
Why isn't the whole text a link?

Reply:
It felt better when looking at it. No real reasons, therefore.